SSO-Auth/Views/apiClient.js

```diff
@@ -0,0 +1,142 @@
+import jellyfinApiclient from "./jellyfin-apiClient.esm.min.js";
+window.jellyfinApiclient = jellyfinApiclient;
+console.log(jellyfinApiclient);
+
+// https://github.com/jellyfin/jellyfin-web/blob/9067b0e397cc8b38635d661ce86ddd83194f3202/src/scripts/clientUtils.js#L19-L76
+export async function serverAddress() {
+  const apiClient = window.ApiClient;
+
+  if (apiClient) {
+    return Promise.resolve(apiClient.serverAddress());
+  }
+
+  const urls = [];
+
+  if (urls.length === 0) {
+    // Otherwise use computed base URL
+    let url;
+    const index = window.location.href.toLowerCase().lastIndexOf("/web");
```

Comment:
The bug is here - 
When served from `http://localhost:8096/subdir/SSOViews/linking`, there's no `/web` on the path. 
I think we need to check for the `SSOViews` path to find the base.

I'll take this as a param

Reply:
Resolved by https://github.com/9p4/jellyfin-plugin-sso/pull/34/commits/a39502d569d8ddc1460d10c11b5c23665c33f816